<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTasks</class>
 <widget class="QDialog" name="AddTasks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_comeback">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Powróć</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>16</y>
     <width>901</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Dodaj zadania</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_title">
        <property name="placeholderText">
         <string>Tytuł</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Wybierz pracownika</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_employees"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_description">
      <property name="placeholderText">
       <string>Opis zadania</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_confirm">
      <property name="text">
       <string>Zatwierdź</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
